### ISSUE TITLE
Update Video component props

### ISSUE THREAD

**Issue by sareh** [bbc/simorgh#1636]
**Is your feature request related to a problem? Please describe.**
Currently the Video component props are outdated - it does not have the necessary inputs. 
https://github.com/bbc/simorgh/blob/latest/src/app/components/Video/index.jsx

**Describe the solution you'd like**
We should have these values passed to the Video component:

**Stats props (From application)**

| Prop | Example | Default |
|------|---------|---------|
| statsCountername | `'news.articles.c000000000o.page'` | | 
| statsDestination | `'NEWS_PS'` or `'NEWS_GNL'` if in UK or outside UK | |
| statsAppType | `'responsive'` when `platform` value in Request context is `'canonical'`.<br /> `'amp'` when `platform` value in Request context is `'amp'` | |
| statsAppName | `'news'` | |
| uiLocale | 'en-GB' | 'en-GB' |

**Video props (From Ares data)**

| Prop | Location in aresMedia block | Example | Default |
|------|-----------|---------|---------|
| pid | model.blocks <br /> find block of type `aresMediaMetadata` <br /> model.id | `p01kdbwr` | |
| kind | model.blocks <br /> find block of type `aresMediaMetadata` <br /> model.subType | `clip` / `radioClip` / `programme` / `radioProgramme` | |
| title | model.blocks <br /> find block of type `aresMediaMetadata` <br /> model.title  |  `'Five things ants can teach us about management'` | |
| items | model.blocks <br /> find block of type `aresMediaMetadata` <br /> model.versions  | array of objects of format: `{ versionID: 'p049sq7k', kind: 'programme', duration: 37}` | |
| holdingImageURL |model.blocks <br /> find block of type `image` <br /> model.blocks <br /> find block of type `rawImage` <br /> model.locator | [1] `'https://ichef.bbci.co.uk/images/ic/$recipe/p049srmr.jpg'` | | 

* [1] Image URL should be of this type, with `$recipe` not interpolated. This is the format that SMP, the media player, expects this URL.

These should be passed from the Article data. 
For `counterName` - using `getPageIdentifier` method in [AnalyticsHelpers Article](https://github.com/bbc/simorgh/blob/latest/src/app/containers/PageViewAnalytics/labelHelpers/article.js)

For `statsDestination` we can use value from [AnalyticsHelpers](https://github.com/bbc/simorgh/blob/latest/src/app/containers/PageViewAnalytics/labelHelpers/index.js)

`statsAppType` should be determined via the Platform Context higher up, in the Video Container. Here it should just be a string value passed in by a prop. 

There should be the correct proptypes defined for the props. 

**Describe alternatives you've considered**
N/A

**Testing notes**
Dev insight: Unit tests should suffice. 

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

**Comment by sareh** [comment]
As a result of discussing the implementation with @Bopchy, we've spun up another issue https://github.com/bbc/simorgh/issues/1636. This issue is now blocked on this work.
